PhysicsTools/PatAlgos/plugins/PATLostTracks.cc

```diff
@@ -235,6 +243,14 @@ void pat::PATLostTracks::addPackedCandidate(std::vector<pat::PackedCandidate>& c
   else if (trkStatus == TrkStatus::PFPOSITRON)
     id = -11;
 
+  // assign the proper pdgId for tracks that are reconstructed as a muon
+  for (auto& mu : *muons) {
+    if (reco::TrackRef(mu.innerTrack()) == trk) {
+      id = -13 * trk->charge();
+      break;
+    }
+  }
+
   reco::Candidate::PolarLorentzVector p4(trk->pt(), trk->eta(), trk->phi(), mass);
```

Comment:
we don't make use of the particle mass. however, if you think it's better to have a consistent mass and pdgId i can add it to the PR

Reply:
I have no preferences here: it depends on the use cases, and also I'd stick on what is done for similar collections in pat.
In your PATMuonMerger you don't rely on the candidate mass of the PATLostTrack, so probably it can stay as such